src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/TestUtils.java

```diff
@@ -241,6 +241,12 @@ public static String getNestedTypeIndexMapping() {
                 "              },\n" +
                 "              \"author\": {\n" +
                 "                \"type\": \"keyword\",\n" +
+                "                \"fields\": {\n" +
```

Comment:
Just observation: I think that we should move index definitions outside of codebase into file system, like it's done with test data (e.g. people2.json and people2.index.json), that will simplify working with those files and importing them into elasticsearch for offline testng

Reply:
> Just observation: I think that we should move index definitions outside of codebase into file system, like it's done with test data (e.g. people2.json and people2.index.json), that will simplify working with those files and importing them into elasticsearch for offline testng

Agree, especially there is no support for multiline string literal in Java at the moment (maybe available in JDK 13?). We can create another issue to improve this (probably as well as a script/gradle task to import our test mapping/data).